### DISCUSSION TITLE
Name change?

### DISCUSSION THREAD

**Discussion by BenKnigge** [go-sqlx/sqlx#8] [Ideas]
What do you think about changing the name? Part of the reason I went with sqly was just to highlight and differentiate it from the forked repo. 

With multiple forks using the existing name it becomes difficult to find the currently maintained fork. If the original author agreed to update the original readme referencing the maintained fork this wouldn't be an issue but as of now he hasn't responded to anyone in over a year.

@uvulpos

**Comment by uvulpos** [comment]
I think maintaining a fork is, in general, a bad idea in terms of SEO and stuff. So what I did intended was to currently stick to a fork, merge open pull requests and later move on to a new name and repository. The advantage of this would be that merging prs from the original source code gets much easier. So, I would decide later 🤔 

What do you think?

**Comment by BenKnigge** [comment]
I think the sooner it has a name the better. 

I personally don't want to have to go through and update dependencies in my existing projects that use sqlx more than once.

**Comment by BenKnigge** [comment]
Once you have the pr's you want to merge, merged, these are the steps to unlink the maintained repo from the original 

https://ralphjsmit.com/unfork-github-repo

The issue with leaving them linked is any new pr is going to want to merge to the original repo by default. It gets confusing.

**Comment by uvulpos** [comment]
Ok, the process to defork the repo got initiated :)
